cabal-install/tests/UnitTests/Distribution/Solver/Modular/DSL.hs

```diff
@@ -1,6 +1,6 @@
+{-# LANGUAGE DuplicateRecordFields #-}
```

Comment:
Mm, how come it adds extensions?

Reply:
As per the advice of HLint, `-XDisambiguateRecordFields` was needed (I'd selected `-XDuplicateRecordFields` by mistake but [this implies](https://ghc.gitlab.haskell.org/ghc/doc/users_guide/exts/duplicate_record_fields.html#extension-DuplicateRecordFields) `-XDisambiguateRecordFields`):

```
$ hlint cabal-install/tests/UnitTests/Distribution/Solver/Modular/DSL.hs
cabal-install/tests/UnitTests/Distribution/Solver/Modular/DSL.hs:3:1-32: 
  Warning: Unused LANGUAGE pragma
Found:
  {-# LANGUAGE RecordWildCards #-}
Perhaps you should remove it.
Note: may require `{-# LANGUAGE DisambiguateRecordFields #-}` adding to the top of the file

1 hint
```

Without it we get `[GHC-76037]` errors:

``` diff
$ git diff
diff --git \
  a/cabal-install/tests/UnitTests/Distribution/Solver/Modular/DSL.hs \
  b/cabal-install/tests/UnitTests/Distribution/Solver/Modular/DSL.hs
index 4f80be0f2..40d10d859 100644
--- a/cabal-install/tests/UnitTests/Distribution/Solver/Modular/DSL.hs
+++ b/cabal-install/tests/UnitTests/Distribution/Solver/Modular/DSL.hs
@@ -1,4 +1,3 @@
-{-# LANGUAGE DuplicateRecordFields #-}
 {-# LANGUAGE GeneralizedNewtypeDeriving #-}
 {-# LANGUAGE OverloadedStrings #-}
 {-# LANGUAGE ScopedTypeVariables #-}
```

```
$ cabal repl cabal-install:unit-tests
...
tests/UnitTests/Distribution/Solver/Modular/DSL.hs:754:7: error: [GHC-76037]
    Not in scope: ‘pkgName’
    Suggested fix:
      Perhaps use record field of PackageIdentifier ‘C.pkgName’ (imported from Distribution.PackageDescription)
    |
754 |     { pkgName = C.mkPackageName (exAvName ex)
    |       ^^^^^^^

tests/UnitTests/Distribution/Solver/Modular/DSL.hs:755:7: error: [GHC-76037]
    Not in scope: ‘pkgVersion’
    Suggested fix:
      Perhaps use one of these:
        record field of PackageIdentifier ‘C.pkgVersion’ (imported from Distribution.PackageDescription),
        record field of PackageDescription ‘C.specVersion’ (imported from Distribution.PackageDescription)
    |
755 |     , pkgVersion = C.mkVersion [exAvVersion ex, 0, 0]
    |       ^^^^^^^^^^

tests/UnitTests/Distribution/Solver/Modular/DSL.hs:769:7: error: [GHC-76037]
    Not in scope: ‘pkgName’
    Suggested fix:
      Perhaps use record field of PackageIdentifier ‘C.pkgName’ (imported from Distribution.PackageDescription)
    |
769 |     { pkgName = C.mkPackageName (exInstName ex)
    |       ^^^^^^^

tests/UnitTests/Distribution/Solver/Modular/DSL.hs:770:7: error: [GHC-76037]
    Not in scope: ‘pkgVersion’
    Suggested fix:
      Perhaps use one of these:
        record field of PackageIdentifier ‘C.pkgVersion’ (imported from Distribution.PackageDescription),
        record field of PackageDescription ‘C.specVersion’ (imported from Distribution.PackageDescription)
    |
770 |     , pkgVersion = C.mkVersion [exInstVersion ex, 0, 0]
    |       ^^^^^^^^^^
...
Failed, 30 modules loaded.
```

Reply:
I fixed the typo (HLS editor selection fumble). The added extension is now `-XDisambiguateRecordFields`.